packages/gatsby-plugin-sharp/README.md

```diff
@@ -96,13 +96,21 @@ a base64 image to use as a placeholder) you need to implement the "blur up"
 technique popularized by Medium and Facebook (and also available as a Gatsby
 plugin for Markdown content as gatsby-remark-images).
 
+When both a `maxWidth` and `maxHeight` are provided, the default way by which the image `fit` is `COVER`, with additional available options of `CONTAIN`, and `FILL`.
+
+`CONTAIN` and `FILL` will not work with `cropFocus` of [sharp.strategy][6]. A `cropFocus` option must be passed that is not `ENTROPY` or `ATTENTION`.
+
+`background` - background colour when using a `fit` of `CONTAIN`, parsed by the [color module][18], defaults to black without transparency.
```

Comment:
I would omit this line as people probably will look it up anyway in sharp if it throws an error

```suggestion
```